### ISSUE TITLE
Blocks#getIdSequence error

### ISSUE THREAD

**Issue by karek314** [LiskHQ/lisk#1149]
```
[inf] 2017-12-14 23:35:49 | Starting sync
[inf] 2017-12-14 23:35:49 | Broadhash consensus now 0 %
[inf] 2017-12-14 23:35:49 | Looking for common block with: 144.217.90.151:8000
[ERR] 2017-12-14 23:35:49 | Error: Unable to set non-blocking to true
    at Client.dispatchQuery (/home/lisk/lisk-main/node_modules/pg-native/index.js:196:26)
    at Client.query (/home/lisk/lisk-main/node_modules/pg-native/index.js:214:8)
    at NativeQuery.submit (/home/lisk/lisk-main/node_modules/pg/lib/native/query.js:128:17)
    at Client._pulseQueryQueue (/home/lisk/lisk-main/node_modules/pg/lib/native/index.js:171:9)
    at Client.query (/home/lisk/lisk-main/node_modules/pg/lib/native/index.js:130:8)
    at /home/lisk/lisk-main/node_modules/pg-promise/lib/query.js:135:27
    at promise (/home/lisk/lisk-main/node_modules/pg-promise/lib/promise.js:22:20)
    at Database.$query (/home/lisk/lisk-main/node_modules/pg-promise/lib/query.js:124:12)
    at Database.<anonymous> (/home/lisk/lisk-main/node_modules/pg-promise/lib/query.js:227:23)
    at /home/lisk/lisk-main/node_modules/pg-promise/lib/database.js:238:42
[ERR] 2017-12-14 23:35:49 | Blocks#getIdSequence error
[ERR] 2017-12-14 23:35:49 | Failed to find common block with: 144.217.90.151:8000
[inf] 2017-12-14 23:35:49 | Looking for common block with: 45.32.102.18:8000
[ERR] 2017-12-14 23:35:49 | Error: Unable to set non-blocking to true
    at Client.dispatchQuery (/home/lisk/lisk-main/node_modules/pg-native/index.js:196:26)
    at Client.query (/home/lisk/lisk-main/node_modules/pg-native/index.js:214:8)
    at NativeQuery.submit (/home/lisk/lisk-main/node_modules/pg/lib/native/query.js:128:17)
    at Client._pulseQueryQueue (/home/lisk/lisk-main/node_modules/pg/lib/native/index.js:171:9)
    at Client.query (/home/lisk/lisk-main/node_modules/pg/lib/native/index.js:130:8)
    at /home/lisk/lisk-main/node_modules/pg-promise/lib/query.js:135:27
    at promise (/home/lisk/lisk-main/node_modules/pg-promise/lib/promise.js:22:20)
    at Database.$query (/home/lisk/lisk-main/node_modules/pg-promise/lib/query.js:124:12)
    at Database.<anonymous> (/home/lisk/lisk-main/node_modules/pg-promise/lib/query.js:227:23)
    at /home/lisk/lisk-main/node_modules/pg-promise/lib/database.js:238:42
[ERR] 2017-12-14 23:35:49 | Blocks#getIdSequence error
[ERR] 2017-12-14 23:35:49 | Failed to find common block with: 45.32.102.18:8000
[inf] 2017-12-14 23:35:49 | Looking for common block with: 94.237.40.140:8000
[ERR] 2017-12-14 23:35:49 | Error: Unable to set non-blocking to true
    at Client.dispatchQuery (/home/lisk/lisk-main/node_modules/pg-native/index.js:196:26)
    at Client.query (/home/lisk/lisk-main/node_modules/pg-native/index.js:214:8)
    at NativeQuery.submit (/home/lisk/lisk-main/node_modules/pg/lib/native/query.js:128:17)
    at Client._pulseQueryQueue (/home/lisk/lisk-main/node_modules/pg/lib/native/index.js:171:9)
    at Client.query (/home/lisk/lisk-main/node_modules/pg/lib/native/index.js:130:8)
    at /home/lisk/lisk-main/node_modules/pg-promise/lib/query.js:135:27
    at promise (/home/lisk/lisk-main/node_modules/pg-promise/lib/promise.js:22:20)
    at Database.$query (/home/lisk/lisk-main/node_modules/pg-promise/lib/query.js:124:12)
    at Database.<anonymous> (/home/lisk/lisk-main/node_modules/pg-promise/lib/query.js:227:23)
    at /home/lisk/lisk-main/node_modules/pg-promise/lib/database.js:238:42
[ERR] 2017-12-14 23:35:49 | Blocks#getIdSequence error
[ERR] 2017-12-14 23:35:49 | Failed to find common block with: 94.237.40.140:8000
[inf] 2017-12-14 23:35:49 | Looking for common block with: 45.32.102.18:8000
[ERR] 2017-12-14 23:35:49 | Error: Unable to set non-blocking to true
    at Client.dispatchQuery (/home/lisk/lisk-main/node_modules/pg-native/index.js:196:26)
    at Client.query (/home/lisk/lisk-main/node_modules/pg-native/index.js:214:8)
    at NativeQuery.submit (/home/lisk/lisk-main/node_modules/pg/lib/native/query.js:128:17)
    at Client._pulseQueryQueue (/home/lisk/lisk-main/node_modules/pg/lib/native/index.js:171:9)
    at Client.query (/home/lisk/lisk-main/node_modules/pg/lib/native/index.js:130:8)
    at /home/lisk/lisk-main/node_modules/pg-promise/lib/query.js:135:27
    at promise (/home/lisk/lisk-main/node_modules/pg-promise/lib/promise.js:22:20)
    at Database.$query (/home/lisk/lisk-main/node_modules/pg-promise/lib/query.js:124:12)
    at Database.<anonymous> (/home/lisk/lisk-main/node_modules/pg-promise/lib/query.js:227:23)
    at /home/lisk/lisk-main/node_modules/pg-promise/lib/database.js:238:42
[ERR] 2017-12-14 23:35:49 | Blocks#getIdSequence error
[ERR] 2017-12-14 23:35:49 | Failed to find common block with: 45.32.102.18:8000
[inf] 2017-12-14 23:35:49 | Looking for common block with: 94.237.64.70:8000
[ERR] 2017-12-14 23:35:49 | Error: Unable to set non-blocking to true
    at Client.dispatchQuery (/home/lisk/lisk-main/node_modules/pg-native/index.js:196:26)
    at Client.query (/home/lisk/lisk-main/node_modules/pg-native/index.js:214:8)
    at NativeQuery.submit (/home/lisk/lisk-main/node_modules/pg/lib/native/query.js:128:17)
    at Client._pulseQueryQueue (/home/lisk/lisk-main/node_modules/pg/lib/native/index.js:171:9)
    at Client.query (/home/lisk/lisk-main/node_modules/pg/lib/native/index.js:130:8)
    at /home/lisk/lisk-main/node_modules/pg-promise/lib/query.js:135:27
    at promise (/home/lisk/lisk-main/node_modules/pg-promise/lib/promise.js:22:20)
    at Database.$query (/home/lisk/lisk-main/node_modules/pg-promise/lib/query.js:124:12)
    at Database.<anonymous> (/home/lisk/lisk-main/node_modules/pg-promise/lib/query.js:227:23)
    at /home/lisk/lisk-main/node_modules/pg-promise/lib/database.js:238:42
[ERR] 2017-12-14 23:35:49 | Blocks#getIdSequence error
[ERR] 2017-12-14 23:35:49 | Failed to find common block with: 94.237.64.70:8000
[ERR] 2017-12-14 23:35:49 | Error: Unable to set non-blocking to true
    at Client.dispatchQuery (/home/lisk/lisk-main/node_modules/pg-native/index.js:196:26)
    at Client.query (/home/lisk/lisk-main/node_modules/pg-native/index.js:214:8)
    at NativeQuery.submit (/home/lisk/lisk-main/node_modules/pg/lib/native/query.js:128:17)
    at Client._pulseQueryQueue (/home/lisk/lisk-main/node_modules/pg/lib/native/index.js:171:9)
    at Client.query (/home/lisk/lisk-main/node_modules/pg/lib/native/index.js:130:8)
    at /home/lisk/lisk-main/node_modules/pg-promise/lib/query.js:135:27
    at promise (/home/lisk/lisk-main/node_modules/pg-promise/lib/promise.js:22:20)
    at Database.$query (/home/lisk/lisk-main/node_modules/pg-promise/lib/query.js:124:12)
    at Database.<anonymous> (/home/lisk/lisk-main/node_modules/pg-promise/lib/query.js:227:23)
    at /home/lisk/lisk-main/node_modules/pg-promise/lib/database.js:238:42
[inf] 2017-12-14 23:35:49 | Broadhash consensus now 0 %
[inf] 2017-12-14 23:35:49 | Finished sync
```

Appeared in one node of mine, stucked in this loop. Rebuild from backup fixed issue. Might be worth looking at it, haven't seen this one yet.

**Comment by karmacoma** [comment]
Thanks @karek314 we will take a look into it.

**Comment by vitaly-t** [comment]
This one is likely to be the answer to all prayers: https://github.com/LiskHQ/lisk/issues/664

**Comment by diego-G** [comment]
It has been reproduced on a node connected to mainnet with `v0.9.12`.  The error arose after stopping the database server while the run was running. When we restart the db server, node gets stuck in that loop. It ends up rebuilding the blockchain. The output is as follows:
```
[ERR] 2018-02-20 11:38:08 | Error: Unable to set non-blocking to true
    at Client.dispatchQuery (/Users/diego-g/git/lisk-0.9/node_modules/pg-
```

A workaround which seems to fix the issue is #664.
